static/app/components/events/eventReplay/replayContent.tsx

```diff
@@ -49,99 +49,78 @@ function ReplayContent({orgSlug, replaySlug, event}: Props) {
   }, [eventTimestamp, startTimestampMs]);
 
   return (
-    <EventDataSection
-      type="replay"
-      title={t('Replay')}
-      actions={
-        <Button
-          size="sm"
-          priority="primary"
-          to={{
-            pathname: `/organizations/${orgSlug}/replays/${replaySlug}/`,
-            query: {
-              t_main: 'console',
-              f_c_search: undefined,
-              ...(initialTimeOffset ? {t: initialTimeOffset} : {}),
-            },
-          }}
-          data-test-id="replay-details-button"
-        >
-          {t('View Details')}
-        </Button>
-      }
-    >
+    <EventDataSection type="replay" title={t('Replay')}>
       {fetching || !replayRecord ? (
         <StyledPlaceholder
           testId="replay-loading-placeholder"
           height="400px"
           width="100%"
         />
       ) : (
-        <table className="table key-value">
-          <tbody>
-            <tr key="replay">
-              <td className="key">{t('Replay')}</td>
-              <td className="value">
-                <ReplayContextProvider
-                  replay={replay}
-                  initialTimeOffset={initialTimeOffset}
+        <ReplayContextProvider replay={replay} initialTimeOffset={initialTimeOffset}>
+          <PlayerContainer ref={fullscreenRef} data-test-id="player-container">
+            <BadgeContainer>
+              <FeatureText>Replays</FeatureText>
```

Comment:
We should probably use `t()` for this label. I checked the sidebar and we're doing it there: https://github.com/getsentry/sentry/blob/master/static/app/components/sidebar/index.tsx#L236


```suggestion
              <FeatureText>{t('Replays')}</FeatureText>
```